src/sci/impl/namespaces.cljc

```diff
@@ -1,7 +1,9 @@
 (ns sci.impl.namespaces
   {:no-doc true}
-  (:require [clojure.string :as str]
-            [clojure.set :as set]))
+  (:refer-clojure :exclude [ex-message])
```

Comment:
@borkdude Is this exclusion needed?

Reply:
Yes, since we are defining a var with the same name.